src/API/useFetch.tsx

```diff
@@ -1,20 +1,27 @@
 //@ts-nocheck
-import React, { useEffect, useState } from "react";
+import { useEffect, useState } from "react";
 import axios from "axios";
 const GITHUB_BASE_URL = 'https://api.github.com';
 
+export interface repositoryCommitMessage  {
+  date: {}
+  commitMessage: "string"
+  url: "string",
+  username: "string"
+} 
+interface repositoryCommitMessages extends Array<repositoryCommitMessage>{}
 
 export const repositoryFetch = async( queryText: string)  => {
   const data = await axios.get(`${GITHUB_BASE_URL}/search/repositories?q=${encodeURIComponent(queryText)}`);
```

Comment:
Let's do some fancy sytax here using less code. 

```js
  const {data}  = await axios.get(`${GITHUB_BASE_URL}/search/repositories?q=${encodeURIComponent(queryText)}`);
  const repositoryChoices = shapeData(data.items, normalizeRepoData)
```

Reply:
no longer using this function